frontends/web/src/routes/buy/pocket.tsx

```diff
@@ -37,12 +38,15 @@ export const Pocket = ({ code }: TProps) => {
   const [height, setHeight] = useState(0);
   const [iframeLoaded, setIframeLoaded] = useState(false);
   const [agreedTerms, setAgreedTerms] = useState(false);
+  const [verifying, setVerifying] = useState(false);
 
   const iframeURL = useLoad(getPocketURL(code));
   const config = useLoad(getConfig);
 
   const ref = createRef<HTMLDivElement>();
   const iframeRef = createRef<HTMLIFrameElement>();
+  var addressType = '';
```

Comment:
this works, but using this var and passing it to the backend could be avoided if backend function simply iterated over all addresses of all script types - could be more robust too, as now it assumes a particular order of calls from the widget.

Reply:
Good point. Now the verification doesn't require the script type as an input